themes/default/content/docs/reference/yaml/_index.md

```diff
@@ -35,11 +35,12 @@ The value of `configuration` is an object whose keys are logical names by which
 
 The value of `resources` is an object whose keys are logical resource names by which the resource will be referenced in expressions within the program, and whose values which are elements of the schema below.  Each item in this object represents a resource which will be managed by the Pulumi program.
 
-| Property        | Type | Required           | Expressions  | Description |
-| ------------- |---|-------------| -----|---|
-| `type`      | string | Yes | No | Type is the Pulumi type token for this resource. |
-| `properties`      | map[string]Expression | No | Yes | Properties contains the primary resource-specific keys and values to initialize the resource state. |
-| `options`         | [Resource Options](#resource-options) | No | No | Options contains all resource options supported by Pulumi. |
+| Property     | Type                                  | Required | Expressions | Description                                                                                         |
+|--------------|---------------------------------------|----------|-------------|-----------------------------------------------------------------------------------------------------|
+| `type`       | string                                | Yes      | No          | Type is the Pulumi type token for this resource.                                                    |
+| `properties` | map[string]Expression                 | No       | Yes         | Properties contains the primary resource-specific keys and values to initialize the resource state. |
+| `options`    | [Resource Options](#resource-options) | No       | No          | Options contains all resource options supported by Pulumi.                                          |
+| `get`        | [Resource Getter](#resource-getter)   | No       | Yes         | A getter function for the resource. Supplying `get` is mutually exclusive to `preoperties`.         |
```

Comment:
```suggestion
| `get`        | [Resource Getter](#resource-getter)   | No       | Yes         | A getter function for the resource. Supplying `get` is mutually exclusive to `properties`.         |
```

Do you also want to mention vice-versa against `properties`?

Reply:
I didn't plan on mentioning it in the description for `properties`. It would make the longest description longer, and you only have to look down 3 columns to see that `get` is mutually exclusive with `properties`.